README.md

```diff
@@ -6,6 +6,20 @@
 
 Static server that hosts [Firefox Account sign up](https://accounts.firefox.com), sign in, email verification, etc. flows.
 
+Table of Contents (ToC)
+=======================
```

Comment:
I don't think we should introduce a second `h1` element at this point, let's remove this heading and just go straight to the links.